# Generated by Django 3.0.7 on 2020-12-20 19:55

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        ('quantumapi', '0018_auto_20201214_1438'),
    ]

    operations = [
        migrations.CreateModel(
            name='ErrorLog',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('environment', models.CharField(blank=True, max_length=100, null=True)),
                ('error', models.TextField(blank=True, null=True)),
                ('component', models.CharField(blank=True, max_length=100, null=True)),
                ('function', models.CharField(blank=True, max_length=100, null=True)),
                ('key', models.TextField(blank=True, null=True)),
                ('session', models.TextField(blank=True, null=True)),
                ('data', models.TextField(blank=True, null=True)),
                ('request_data', models.TextField(blank=True, null=True)),
                ('headers', models.CharField(blank=True, max_length=500, null=True)),
                ('host_ip', models.CharField(blank=True, max_length=100, null=True)),
                ('date', models.DateTimeField(default=django.utils.timezone.now)),
                ('user', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
            options={
                'verbose_name': 'error log',
                'verbose_name_plural': 'error logs',
                'ordering': ('user',),
            },
        ),
    ]
